packages/api/src/external/fhir/document/index.ts

```diff
@@ -144,3 +156,110 @@ export function idToFHIR(id: DocumentIdentifier): Identifier {
     use: id.use === "unspecified" ? undefined : id.use,
   };
 }
+
+/**
+ * Takes a CW resource and converts it to a FHIR resource
+ *
+ * @param resource
+ * @returns
+ */
+function convertToFHIRResource(resource: Contained): Resource | undefined {
+  if (resource.resourceType === ResourceType.Patient && resource.id) {
+    return {
+      resourceType: ResourceType.Patient,
+      id: resource.id,
+    };
+  } else if (resource.resourceType === ResourceType.Organization && resource.name) {
+    return {
+      resourceType: ResourceType.Organization,
+      name: convertCWNameToString(resource.name),
+    };
```

Comment:
No other properties to include? IIRC, its common to have those contained resources identified w/ a reference of sorts, which are then used throughout the doc ref to "link" to this resource.

Reply:
Yeah I was going to comment the same thing. So we're on the same page, look at this doc query result from CW: https://drive.google.com/file/d/1wb8xGOW7qeKQIbybkvSnCffry_8VVkjW/view?usp=drive_link

Consider this example:
```
    {
      "id": "urn:uuid:69821b0e-REDACTED",
      "content": {
        "resourceType": "DocumentReference",
        "contained": [
          {
            "resourceType": "Organization",
            "id": "orgRef9",
            "identifier": [{ "value": "1.2.840.REDACTED" }],
            "name": "UCSF Health, UCSF Children's, Marin Health, and Affiliates"
          },
          {
            "resourceType": "Practitioner",
            "id": "authRef9",
            "name": {
              "family": ["UCSF Health, UCSF Children's, Marin Health, and Affiliates"],
              "given": [""],
              "prefix": [""]
            },
            "organization": { "reference": "#orgRef9" }
          }
        ],
        ...
        "subject": { "reference": "#patRef9" },
        "type": {
          "coding": [
            { "system": "http://loinc.org/", "code": "11506-3", "display": "Hospital Encounter" }
          ]
        },
        "author": [{ "reference": "#authRef9" }],
         ...
      }
    },
```

We don't want to just throw away the id/identifier of the contained objects, as they're useful, and referenced in other parts of the doc ref.

Also, we need to fix this https://github.com/metriport/metriport/blob/b6d0eb0e9253c47556363941e768ff763a69ec39/packages/api/src/external/fhir/document/index.ts#L94-L99 - the author is not the Metriport organization making the query, it should be referring to the org in contained if that makes sense